models/articleModel.js

```diff
@@ -0,0 +1,24 @@
+const db = require('../db/connection.js');
+
+exports.fetchArticles = (article_id) => {
+    if (isNaN(parseInt(article_id))){
+        return Promise.reject({
+            status: 400,
+            msg: 'bad request - invalid data format',
+        });
+    };
+
+    return db.query(`
+        SELECT * FROM articles
+        WHERE article_id = $1;
+    `, [article_id])
+    .then((data)=>{
+        if (data.rowCount === 0) {
+            return Promise.reject({
+              status: 404,
+              msg: "Not Found",
+            });
+        }
+        return data;
```

Comment:
If we look at data before you return it, you'll know the information you want is on the key of rows. Here you could return the first element in that array to make your controller neater.